packages/components/src/presentation-components/tree/controlled/PresentationTreeNodeRenderer.tsx

```diff
@@ -22,6 +22,7 @@ import { isPresentationInfoTreeNodeItem, isPresentationTreeNodeItem, Presentatio
 export interface PresentationTreeNodeRendererProps extends TreeNodeRendererProps {
   onFilterClick: (node: PresentationTreeNodeItem) => void;
   onClearFilterClick: (node: PresentationTreeNodeItem) => void;
+  getTreeModel?: () => TreeModel;
```

Comment:
Maybe `onFilterClick` and `onClearFilterClick` can take `nodeId: string` instead of `node: PresentationTreeNodeItem`? That way you won't need `getTreeModel` and can just call `onFilterClick(node.parentId)`

Reply:
Good call, will change it